crates/epaint/src/tessellator.rs

```diff
@@ -868,6 +866,21 @@ fn fill_closed_path_with_uv(
     }
 }
 
+/// Translate a point according to the stroke kind.
+fn translate_stroke_point(p: &PathPoint, stroke: &PathStroke) -> PathPoint {
+    match stroke.kind {
+        stroke::StrokeKind::Middle => p.clone(),
```

Comment:
nit: we should add `#[derive(Copy)]` on `PathPoint` so it is obvious that this is cheap

```suggestion
        stroke::StrokeKind::Middle => *p,
```